### DISCUSSION TITLE
Bosch CookIt also supported?

### DISCUSSION THREAD

**Discussion by nanosonde** [ekutner/home-connect-hass#79] [Community Support] [answered]
Hi!
We are using the Bosch CookIt cooking processor at home. It also uses the Home Connect stuff to enable basic cooking status info (like current recipe, recipe step and remaining time in the step) in the Home Connect app. It also allows the device to be stopped (but not started) from the app.

Is this supported by this integration?

**Comment by ekutner** [comment]
Did you try it? There is nothing in the integration that is specific to any appliance, if the appliance is supported by the Home COnnect API then it should work.

**Comment by nanosonde** [comment, marked as answer]
Seems to work:

![grafik](https://user-images.githubusercontent.com/2073569/180454334-15c31516-9cd5-493e-a445-2d0f4a94bc09.png)

**Reply by ChristophCaina** [reply]
it seems that some of the options / enum types are missing in the translation files... ?
Not sure, if this is already fixed...

if not, a list of the options and values might be helpful to add them :)
